improver/cli/threshold.py

```diff
@@ -157,11 +163,15 @@ def process(
         raise ValueError("Cannot apply land-mask cube without in-vicinity processing")
 
     if collapse_coord == "realization":
-        # TODO change collapse_coord argument to boolean "collapse_realizations"
-        # (requires suite change)
+        each_threshold_func_list.append(collapse_realizations)
+    elif collapse_coord == "percentile":
+        cube = RebadgePercentilesAsRealizations()(cube)
```

Comment:
I think that the `RebadgePercentilesAsRealizations` percentile checking is slightly stricter than @bayliffe was suggesting is required. I think that the percentile set [10, 30, 50, 70, 90] would run into an error if we tried to rebadge as percentiles because it doesn't equally partition percentile space. 

This isn't necessarily a problem which we'll run into though, depending on which percentiles @gavinevans draws from the calibrated windspeeds distribution.

Reply:
Yep, Gavin and I chatted about this. Gavin is intending to produce an evenly spaced set from which the product percentiles are drawn / generated later. So it is stricter than I suggested, but will work for Gavin's purposes and I'm happy to make it more rigorous.

Reply:
Yes, for thresholding purposes, the [10, 30, 50, 70, 90] percentile set isn't necessarily incorrect, as far as I can see. For our immediate purposes, this isn't an issue as the intention is to use a percentile set that is evenly spaced around the 50th percentile, however, you're correct that we're probably being stricter than necessary, in terms of what percentiles are acceptable for conversion to probabilities. Perhaps we can consider this in slower time?